src/util/segment.js

```diff
@@ -35,8 +35,8 @@ export const probeTsSegment = (bytes, baseStartTime) => {
 
   const result = {
     // each type's time info comes back as an array of 2 times, start and end
-    hasVideo: timeInfo.video && timeInfo.video.length === 2,
-    hasAudio: timeInfo.audio && timeInfo.audio.length === 2
+    hasVideo: timeInfo.video && timeInfo.video.length === 2 || false,
```

Comment:
why is this needed?

Reply:
previously this will result in `hasVideo` and `hasAudio` being set to undefined, rather than a `boolean` value of `false`.

Reply:
Oh, if `timeInfo.video` was undefined, `hasVideo` would be undefined? Gotcha, makes more sense.